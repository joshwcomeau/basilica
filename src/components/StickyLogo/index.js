// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';
import { Sticky } from 'react-sticky';

import Logo from '../Logo';

import './index.scss';


const StickyLogo = () => {
  const classes = classNames(['sticky-logo']);

  return (
    <Sticky
      className={classes}
      style={{
        position: 'absolute',
        height: '50px',
        lineHeight: '50px',
        top: '-55px',
        left: '15px',
      }}
      stickyStyle={{
        width: 'auto',
        left: '25px',
      }}
      topOffset={-55}
    >
      <Logo />
    </Sticky>
  );
};

StickyLogo.propTypes = {

};

StickyLogo.defaultProps = {

};

export default StickyLogo;
