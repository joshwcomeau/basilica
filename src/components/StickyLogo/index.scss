@import '../../variables';

.sticky-logo {

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
