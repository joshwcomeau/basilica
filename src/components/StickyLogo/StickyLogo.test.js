/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import StickyLogo from './index';

it('renders correctly', () => {
  const tree = renderer.create(<StickyLogo />).toJSON();
  expect(tree).toMatchSnapshot();
});
