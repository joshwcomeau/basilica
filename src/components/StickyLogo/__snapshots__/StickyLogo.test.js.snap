exports[`test renders correctly 1`] = `
<div
  className="sticky-logo">
  Your Component Here :)
</div>
`;
